examples/example-snippets/process_management.cf

```diff
@@ -155,21 +156,21 @@ bundle agent process_restart
 
 body common control
 {
-      bundlesequence  => { "testbundle"  };
+    bundlesequence => { "testbundle" };
 }
 ########################################################
 
 bundle agent testbundle
 {
   processes:
 
-      "cfservd"
-      process_count   => up("cfservd");
+    "cfservd"
+      process_count => up("cfservd");
 
     cfservd_out_of_control::
       "cfservd"
-      signals         => { "stop" , "term" },
-      restart_class   => "start_cfserv";
+        signals       => { "stop" , "term" },
+        restart_class => "start_cfserv";
```

Comment:
Can you alter this `start_cfserv` to `stopped_out_of_control_cfserv` while your here. It's bad practice IMHO to define classes that represent the /desired/ state. Instead, I think it's better to define classes that represent the current state. It makes policy read more easily:

e.g.

```
cfserved_out_of_control::
  signals => { "stop", "term" },
  restart_class => "stopped_cfserved_out_of_control";

commands:
  stopped_cfserved_out_of_control;
    "/var/cfenigne/bin/cf-serverd";

```

Reply:
Sure: https://github.com/cfengine/documentation/pull/2384